extension/e2e-tests/helpers/login.ts

```diff
@@ -0,0 +1,32 @@
+import { generateMnemonic } from "stellar-hd-wallet";
+import { expect } from "../test-fixtures";
+
+export const PASSWORD = "My-password123";
+
+export const login = async ({ page, extensionId }) => {
+  await page.goto(`chrome-extension://${extensionId}/index.html`);
+  await page.getByText("Import Wallet").click();
+
+  const TEST_WORDS = generateMnemonic({ entropyBits: 128 }).split(" ");
+
+  for (let i = 1; i <= TEST_WORDS.length; i++) {
+    await page.locator(`#MnemonicPhrase-${i}`).fill(TEST_WORDS[i - 1]);
+  }
+
+  await page.locator("#password-input").fill(PASSWORD);
+  await page.locator("#confirm-password-input").fill(PASSWORD);
+  await page.locator("#termsOfUse-input").check({ force: true });
+  await page.getByRole("button", { name: "Import" }).click();
+  await expect(page.getByText("Wallet created successfully!")).toBeVisible({
+    timeout: 20000,
+  });
+
+  await page.goto(`chrome-extension://${extensionId}/index.html#/account`);
+  await page.getByTestId("network-selector-open").click();
+  await page.getByText("Test Net").click();
+  await page.getByRole("button", { name: "Fund with Friendbot" }).click();
+
+  await expect(page.getByTestId("account-assets")).toBeVisible({
+    timeout: 10000,
```

Comment:
Are these timeouts to wait for the network? I wonder if that would make them flaky at times?
Is there something else we can wait for, like maybe an `await page.getByText("<Some title after the page loads?>")`
Maybe this is fine if the timeout is long enough also, 10 seconds should cover long tail latency I would think.

Reply:
Yeah, this was to deal with Horizon requests sometimes taking a long time to resolve. This one, for example, is waiting for the above `Fund With Friendbot` action to complete and for the assets to actually populate on screen.

I had run into a couple instances of the tests timing out before Horizon would come back with a response. I wanted to keep the timeout pretty long to give the network request plenty of time to resolve. If it takes longer than 10 seconds, it may be a sign that something is up with Horizon